dev/preview/util/gcloud-login.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+if [[ $(gcloud auth list --format=json) == '[]' ]]; then
+    echo
+    echo "Currently you need to be authenticated with gcloud to run the build"
+    echo
+    echo "This is needed to you can use the Leeway build cache that Werft also uses"
+    echo "We're working on automating this here: https://github.com/gitpod-io/gitpod/issues/13714"
+    echo
+    echo "But for now you have to run the following log in manually"
+    echo
+    gcloud auth login --no-launch-browser
+    echo
+    echo "Great, thanks!"
+    echo
+    echo "Continuing the build"
+fi
```

Comment:
Is this file needed for this PR? I'm asking because it looks like an unrelated change and potential copy of 
https://github.com/gitpod-io/gitpod/blob/721dd00364add7973e8064bb70eff614cda00de3/dev/preview/workflow/lib/ensure-gcloud-auth.sh#L7-L19

Reply:
I needed it to speed up testing of supervisor in prev envs with hot deploy scrip. Without login remote caches are not used and I had to wait for everything to be rebuilt locally. After login only supervisor image is rebuilt.

Reply:
ok, then let's keep it. We're can refactor it later as part of our work on https://github.com/gitpod-io/ops/issues/5745.